avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -109,14 +109,178 @@ def getRefMB(testName, pathDict, simName):
 
     # Get info from ExpLog
     mbFile = os.path.join('..', 'benchmarks', testName, 'mass_%s.txt' % simName)
-    pathDict[simName]['mb'].append(mbFile)
+    pathDict['mb'].append(mbFile)
     log.info('Added to pathDict[mb] %s' % (mbFile))
 
     return pathDict
 
 
-def dfaComp2Aimec(avaDir, cfgSetup):
-    """ Exports the required data from com1DFA to be used by Aimec """
+def dfaComp2Aimec(avaDir, cfg, rel, simType):
+    """ Create a pathDict where the paths to all the files required by aimec are saved for two modules -
+        in order to compare always two simulations at a time
+
+        for now matchin simulations are identified via releaseScenario and simType
+
+        Parameters
+        -----------
+        avaDir: str
+            path to avalanche directory
+        cfgSetup: configParser object
+            configuration for aimec
+        rel: str
+            releaseScenario
+        simType: str
+            simulation type (null, ent, entres, ..)
+
+        Returns
+        --------
+        pathDict: dict
+            dictionary with paths to result and optionally mass files for matching simulations from two modules - matching in terms
+            of releaseScenario and simType
+    """
+
+    cfgSetup = cfg['AIMECSETUP']
+
+    # initialise empty pathDict for all required files
+    pathDict = {'ppr': [], 'pfd': [], 'pfv': [], 'mb': []}
+    # get directories where simulation results can be found for both modules
+    inputDirRef, inputDirComp, pathDict, refModule = getCompDirs(avaDir, cfgSetup, pathDict)
+
+    # Load all infos on reference simulations
+    if refModule == 'benchmarkReference':
+        refData = fU.makeSimDict(inputDirRef)
+    else:
+        refData = fU.makeSimDict(inputDirRef)
+
+    # Load all infos on comparison module simulations
+    compData = fU.makeSimDict(inputDirComp)
+
+    # identify names of simulations that match criteria for both cases
+    simSearch = True
+    for countRef, simNameShort in enumerate(refData['simName']):
+        for countComp, simNameComp in enumerate(compData['simName']):
+            if simSearch:
+                if refData['releaseArea'][countRef] == compData['releaseArea'][countComp] == rel and refData['simType'][countRef] == compData['simType'][countComp] == simType:
+                    refSimName = refData['simName'][countRef]
+                    compSimName = compData['simName'][countComp]
+                    log.info('Reference simulation: %s and to comparison simulation: %s ' % (refSimName, compSimName))
+                    simSearch = False
```

Comment:
maybe there's a better comparison solution with dict/list comparison tools?

If not, at least use a `while` loop. Since once `simSearch` is False, it keeps looping over both for loops...

Reply:
this could be removed, if we chose the new option to filter sims using the configuratioFiles and a dataFrame - this is shown in runStandardTestsPySP.py